Python/bltinmodule.c

```diff
@@ -2649,9 +2649,14 @@ static PyMethodDef zip_methods[] = {
 };
 
 PyDoc_STRVAR(zip_doc,
-"zip(*iterables) --> zip object\n\
+"zip(*iterables) --> An iterable zip object that yields tuples.\n\
```

Comment:
To make the single line help text more useful, I'm pondering changing this to

`A zip object that yields tuples until one input is exhausted.`